elastic/integrations
# updated max connections to max agent

## What does this PR do?

Converts the max connections to a max agents field. This change is related to https://github.com/elastic/fleet-server/pull/1014.

Technically a breaking change, since this removes a field that may be used by users. 

## Checklist


## How to test this PR locally

Host the package registry, enroll and run an agent, set a value in the Max agents field and observe the config come down as such:

```
...
    inputs:
    - data_stream:
        namespace: default
      id: default-fleet-server-agent-policy
      meta:
        package:
          name: fleet_server
          version: 2.0.0
      name: fleet_server-1
      revision: 3
      server:
        host: 0.0.0.0
        limits.max_agents: 100
        port: 8220
      type: fleet-server
      use_output: default

...
```

## Related issues

Closes https://github.com/elastic/fleet-server/issues/571
